packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -288,13 +288,10 @@ Future<void> _runTests(List<String> testArgs, String observatoryUri) async {
   PackageMap.globalPackagesPath = fs.path.normalize(fs.path.absolute(PackageMap.globalPackagesPath));
   final String dartVmPath = fs.path.join(dartSdkPath, 'bin', 'dart');
   final int result = await runCommandAndStreamOutput(
-    <String>[dartVmPath]
-      ..addAll(dartVmFlags)
-      ..addAll(testArgs)
-      ..addAll(<String>[
+    <String>[dartVmPath, ...dartVmFlags, ...testArgs, ...<String>[
```

Comment:
This is so cool!  Again though, either turning `[stuff]..addAll([elements])` into `[stuff, elements]` or else having a separate quick fix for `[...[elements]]` to `[elements]` would be great.